SEP-0009.md

```diff
@@ -0,0 +1,69 @@
+# SEP-0009 - Astropy Time
+
+| SEP           | 9                                         |
+|---------------|-------------------------------------------|
+| title         | Astropy Time                              |
+| author(s)     | Stuart Mumford                            |
+| contact email | stuart@cadair.com                         |
+| date-creation | 2018-05-30                                |
+| type          | standard                                  |
+| status        | discussion                                |
+| discussion    | https://github.com/sunpy/sunpy/issues/993 |
+
+# Introduction
+
+This SEP proposes that SunPy transitions all internal, and external use of a
+Python object to represent a date or time from the standard library `datetime`
+module to the classes provided in `astropy.time`. The main classes being
+replacing `datetime.date` and `datetime.datetime` with `astropy.Time` and
+replacing `datetime.timedelta` with `astropy.time.TimeDelta`.
+
+# Detailed Description
+
+Since its inception the SunPy core library has used `datetime` for handling
+time. This is a very useful standard library module, however it lacks some key
+benefits that are useful to the solar community:
+
+- Support for non-UTC time scales. This is especially useful for JSOC and HMI data which use the TAI time scale rather than UTC. 
+- Support for high (double-double) precision time representation and arithmetic.
+- Support for leap seconds.
+- Support for different representations of time, i.e. Julian Day, unix time, GPS.
+- Support for reading and writing the new FITS Time standard.
+- Support for light travel time calculations for different locations on Earth.
+- Support for custom Time scales and formats, i.e. utime.
+
+
+## parse_time
+
+All over SunPy when time-like objects are accepted as input, this input is
+sanitised with the `sunpy.time.parse_time` function. This function currently
```

Comment:
Add a sentence here mandating that parse_time should be used to sanitize all time input.